mobile/native/src/ln_dlc/channel_status.rs

```diff
@@ -104,100 +93,63 @@ async fn channel_status(node: impl Borrow<Node>) -> Result<ConcreteChannelStatus
 #[derive(Debug, Clone, Copy)]
```

Comment:
The docs should now read: "The complete status of the app's DLC channel". It talked about channel**S** before because we had LN and DLC channels.